src/js/controllers/index.js

```diff
@@ -935,8 +936,10 @@ angular.module('copayApp.controllers').controller('indexController', function($r
       });
     });
   }
+
   self.showAllHistory = function() {
     self.historyShowShowAll = false;
+    self.shouldHideShowAll = true;
```

Comment:
why there are two variables:

 - historyShowShowAll
and
 - shouldHideShowAll 

isn't that the same?